### PR TITLE
fix: [M3-8686] - fix Buttons data-qa attributes

### PR DESCRIPTION
**Pull request by abailly-akamai**
## Description 📝
Fixing a console error that has bothered me for a long time!

Screenshot 2024-10-02 at 10 43 14

## Changes  🔄
- fix `ActionPanel` `PrimaryButton` `data-qa` attribute in `SingleTextfieldForm`
- updates unit and e2e tests

### Reproduction steps
- navigate to and reload `/profile/display`
- notice console error

### Verification steps
- navigate to and reload `/profile/display`
- confirm console error gone
- confirm unit and e2e suites are happy

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/components/SingleTextFieldForm/SingleTextFieldForm.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ export const SingleTextFieldForm = React.memo((props: Props) => {
         />
         <ActionsPanel
           primaryButtonProps={{
-            'data-testid': 'Button',
+            'data-testid': 'button',
```
Comment (abailly-akamai) [inline packages/manager/src/components/SingleTextFieldForm/SingleTextFieldForm.tsx:96]: This is the main change here, that was giving us the console error

**Review by jdamore-linode — APPROVED** [review]
🙌 Thanks @abailly-akamai!

I never noticed the console error, but I _have_ noticed the random uppercased "Button" test ID -- happy to see this change!

File: packages/manager/src/components/Button/Button.tsx [hunk 2]
```diff
@@ -158,7 +158,7 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         color={color}
         compactX={compactX}
         compactY={compactY}
-        data-testid={rest['data-testid'] || 'Button'}
+        data-testid={rest['data-testid'] || 'button'}
```
Comment (abailly-akamai) [inline packages/manager/src/components/Button/Button.tsx:161]: This is 1/2 of the main changes

**Review by bnussman-akamai — APPROVED** [review]
🙏 🚀

**Pull request merged**